Service/ExpressPay/Order/Update.php

```diff
@@ -95,12 +94,17 @@ public function execute($quoteMaskId, $gatewayId, $paypalOrderId): void
         $uri = "checkout/orders/{{shopId}}/wallet_pay/$paypalOrderId";
         $expressPayData = $this->quoteConverter->convertFullQuote($quote, $gatewayId);
 
-        $expressPayOrder = $this->getExpressPayOrder->execute($paypalOrderId, $gatewayId);
-        $expressPayOrderShipping = $expressPayOrder['data']['shipping_address'];
+        try {
+            $expressPayOrder = $this->getExpressPayOrder->execute($paypalOrderId, $gatewayId);
+        } catch (LocalizedException $localizedException) {
+            $expressPayOrder = [];
+        }
+
+        $expressPayOrderShipping = $expressPayOrder['shipping_address'] ?? [];
```

Comment:
I believe there should be a `['data']` level in there as well `$expressPayOrder['data']['shipping_address']`

Reply:
Scratch that - Just realized you modified the return from the get

Reply:
Yes, the `data` key was superfluous since it's the only key in the array, so I modified the Get service to not return it.